blazor-wasm/wwwroot/index.html

```diff
@@ -0,0 +1,73 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>WebApplication1</title>
+    <base href="/" />
+    <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
+    <link href="css/site.css" rel="stylesheet" />
+    <link href="css/index.css" rel="stylesheet" />
+</head>
+<body>
+    <app>Loading...</app>
+
+    <script src="_framework/blazor.webassembly.js"></script>
+
+    <script src="charting_library/charting_library.min.js"></script>
+    <script src="datafeeds/udf/dist/bundle.js"></script>
+
+    <script>
+        function getLanguageFromUrl() {
+            const regex = new RegExp('[\\?&]lang=([^&#]*)');
+            const results = regex.exec(window.location.search);
+            return results === null ? null : decodeURIComponent(results[1].replace(/\+/g, ' '));
+        };
+
+        var tvWidget = null;
+        window.tradingViewChartWrapper = {
+
+            initialize: function () {
+                const widgetOptions = {
+                    symbol: 'AAPL',
+                    // BEWARE: no trailing slash is expected in feed URL
+                    datafeed: new window.Datafeeds.UDFCompatibleDatafeed('https://demo_feed.tradingview.com'),
+                    interval: 'D',
+                    container_id: 'tv_chart_container',
+                    library_path: '/charting_library/',
+                    locale: getLanguageFromUrl() || 'en',
+                    disabled_features: ['use_localstorage_for_settings'],
+                    enabled_features: ['study_templates'],
+                    charts_storage_url: 'https://saveload.tradingview.com',
+                    charts_storage_api_version: '1.1',
+                    client_id: 'tradingview.com',
+                    user_id: 'public_user_id',
+                    fullscreen: false,
+                    autosize: true,
+                    studies_overrides: {}
+                };
+
+                tvWidget = new TradingView.widget(widgetOptions);
+
+                tvWidget.onChartReady(() => {
+                    tvWidget.headerReady().then(() => {
+                        const button = tvWidget.createButton();
+                        button.setAttribute('title', 'Click to show a notification popup');
+                        button.classList.add('apply-common-tooltip');
+                        button.addEventListener('click',
+                            () => tvWidget.showNoticeDialog({
+                                title: 'Notification',
+                                body: 'TradingView Charting Library API works correctly',
+                                callback: () => {
+                                    console.log('Noticed!');
+                                }
+                            }));
+
+                        button.innerHTML = 'Check API';
+                    });
+                });
+            }
+        };
```

Comment:
A better place to store the initialization script is a separate javascript file.